_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: distill
+title: "Lecture 8: Causal Discovery and Inference"
+description: <insert description here>
+date: <insert date here>
+
+lecturers:
+  - name: Eric Xing
+    url: "https://www.cs.cmu.edu/~epxing/"
+
+authors:
+  - name: Abhilasha Ravichander
+    url: "#"  # optional URL to the author's homepage
+  - name: Aniruddh Shetty
+    url: "#"
+  - name: Tom Yan
+    url: "#"
+  - name: Kin Gutierrez Olivares
+
+editors:
+  - name: Lisa Lee
+    url: "http://leelisa.com"
+
+abstract: >
+---
+Causation implies correlation (or dependence), but correlation does not imply causation. Consider the case of chocolate consumption vs. the number of Nobel Laureates in the country:
+
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe1.jpg' | relative_url }}"  width="600" height="400" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Chocolate Consumption vs. # of Nobel Laureates</strong>
+  </figcaption>
+</figure>
+
+From this plot, we can see that they are highly correlated. But it is absurd to say that if we want to increase the number of Nobel Laureates in a country, we need to eat more chocolate. This is where the idea of causality comes in. They are correlated, but chocolate consumption is not the cause of Nobel Laureates. The following equations capture this difference mathematically:
+
+$$X$$ and $$Y$$ are dependent if and only if $$ \exists x_1 $$ different than $$ x_2 $$ such that:
+
+$$
+P(Y \mid X=x_{1}) \text{ different than } P(Y \mid X=x_2)
+$$
+
+$$X$$ is a cause of $$Y$$ if and only if $$ \exists x_1 $$ different than $$ x_2 $$ such that:
+
+$$
+P(Y \mid do(X=x_1) \text{ different than } P(Y \mid X=x_2) 
+$$
+
+The definition requires just a pair of distinct $$ x_1, x_2 $$ to have different conditional distribution of Y. Y might have same conditional distribution over a range of x values. But if we find even a single pair for which the conditional distribution is different, X and Y are dependent. Same goes for causation 
```

Comment:
As requested in the previous review, please change
- "X" to "$$X$$"
- "Y" to "$$Y$$"
everywhere in the file.